holidays/countries/micronesia.py

```
@@ -0,0 +1,209 @@
+        self._add_observed(self._add_holiday_nov_8(tr("Pohnpei Constitution Day")))
+
+    # Yap.
+    def _populate_subdiv_yap_public_holidays(self):
+        # Yap Day.
+        self._add_observed(self._add_holiday_mar_1(tr("Yap Day")))
+
+        # Yap State Constitution Day.
+        self._add_observed(self._add_holiday_dec_24(tr("Yap State Constitution Day")))
+
+
+class FM(Micronesia):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)